### ISSUE TITLE
Update for IPTV channels for mana2

### ISSUE THREAD

**Issue by MobCode100** [weareblahs/freeview#44]
IPTV channels from mana2 now requires `auth_key` and `token` to be passed as URL parameters. E.g. 
`https://live.mana2.my/SukeTv/index.m3u8?auth_key=1720818213-02370a51bf91417c87af11cf340f16a6-0-99134be278ebfd76c1dacc1848176654&token=1720818213-02370a51bf91417c87af11cf340f16a6-0-99134be278ebfd76c1dacc1848176654`

I think those parameters do not have any expiry. Not that I've tested this, but those parameters could be anything obtained from the web application of MYTV.

**Comment by PRSXFENG** [comment]
Yeah the same also applies for Tonton, and also Bernama radio, but so far we dont really want to put tokens in there since it could be traced to a user
